Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -79,9 +110,12 @@ def convert_date_to_timestamp(date):
     if TIME_METHOD == 'Timestamp-Seconds':
         return int(date.timestamp())
 
-    else:  # this case includes "Timestamp-Milliseconds"
+    elif TIME_METHOD == 'Timestamp-Milliseconds':
         return int(date.timestamp() * 1000)
 
+    else:  # In case of 'Simple-Date'.
```

Comment:
You returned above, so you don't need `else` here, as this is the default return value.